restcomm/restcomm.http/src/main/java/org/restcomm/connect/http/SupervisorEndpoint.java

```diff
@@ -184,7 +187,7 @@ protected Response registerForUpdates(final String accountSid, final Multivalued
     //Register a remote location where Restcomm will send monitoring updates for a specific Call
     protected Response registerForCallUpdates(final String accountSid, final String callSid, final MultivaluedMap<String, String> data, MediaType responseType) {
         secure(daos.getAccountsDao().getAccount(accountSid), "RestComm:Read:Calls");
```

Comment:
very good point @otsakir I think i would agree that following 2 things are enought to grant access:
1. a valid toke is present
2. it is a super admin.

Since we agree here so letme edit the code

Thanks a lot for your valuable feedback.